content-src/components/Snippet/Snippet.js

```diff
@@ -0,0 +1,37 @@
+const React = require("react");
+const classNames = require("classnames");
+
+const Snippet = React.createClass({
```

Comment:
Sure, but along with the comment, please do the PureRenderMixin thing now -- it's a one-liner:

https://github.com/mozilla/loop/blob/616b1e9c2dd844a6068cd194ac6d17d771241981/add-on/panels/js/desktopViews.jsx#L19